draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1391,7 +1391,7 @@ Steps:
 ### Little endian variant {#sgn0-le}
 
 The following sgn0 variant is defined such that sgn0\_le(x) = -1
-just when x != 0 and the parity of the least significant nonzero
+just when x != 0 and the parity of the smallest-degree nonzero
```

Comment:
This change should probably be reverted too, right?

Reply:
true.